db/mainphotos copy.sql

```diff
@@ -0,0 +1,74 @@
+CREATE SCHEMA IF NOT EXISTS mainphotos;
+
+CREATE TABLE mainphotos.neighborhoods (
+  neighborhood_id INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
```

Comment:
you can use this shorthand: `neighborhood_id **SERIAL** PRIMARY KEY,`
serial keyword does:
- auto increment
- add INT constraint
- add NOT NULL constraint
- assigns the owner of the sequence to the id column; as a result, the sequence object is deleted when the id column or table is dropped